src/methodical/impl/combo/common.clj

```diff
@@ -1,5 +1,91 @@
 (ns methodical.impl.combo.common
-  "Utility functions for implementing method combinations.")
+  "Utility functions for implementing method combinations."
+  (:require [methodical.util :as u]))
+
+(defn partial*
+  "[[clojure.core/partial]] but with more direct arities."
```

Comment:
It would be nice to add a Kondo `:discouraged-var` rule for `clojure.core/partial` but I can do that separately